<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblRegionalCode">
     <property name="text">
      <string>&amp;Региональный код</string>
     </property>
     <property name="buddy">
      <cstring>edtRegionalCode</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="edtRegionalCode"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>&amp;Наименование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1" rowspan="2" colspan="2">
    <widget class="CInDocTableView" name="tblAnalogs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblAnalogs">
     <property name="text">
      <string>Аналоги</string>
     </property>
     <property name="buddy">
      <cstring>tblAnalogs</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbClass"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblClass">
     <property name="text">
      <string>&amp;Класс</string>
     </property>
     <property name="buddy">
      <cstring>cmbClass</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblKind">
     <property name="text">
      <string>&amp;Вид</string>
     </property>
     <property name="buddy">
      <cstring>cmbKind</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblType">
     <property name="text">
      <string>&amp;Тип</string>
     </property>
     <property name="buddy">
      <cstring>cmbType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbKind"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbType"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblProducer">
     <property name="text">
      <string>&amp;Производитель</string>
     </property>
     <property name="buddy">
      <cstring>edtProducer</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblATC">
     <property name="text">
      <string>Код &amp;АТХ</string>
     </property>
     <property name="buddy">
      <cstring>edtATC</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblPackSize">
     <property name="text">
      <string>&amp;Объём упаковки</string>
     </property>
     <property name="buddy">
      <cstring>spnPackSize</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="edtProducer"/>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="edtATC"/>
   </item>
   <item row="8" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="spnPackSize">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbClass</tabstop>
  <tabstop>cmbKind</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtRegionalCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtProducer</tabstop>
  <tabstop>edtATC</tabstop>
  <tabstop>spnPackSize</tabstop>
  <tabstop>tblAnalogs</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
